Doc/library/wsgiref.rst

```diff
@@ -754,6 +754,49 @@ input, output, and error streams.
    .. versionadded:: 3.2
 
 
+:mod:`wsgiref.types` -- WSGI types for static type checking
+-----------------------------------------------------------
+
+.. module:: wsgiref.types
+   :synopsis: WSGI types for static type checking
+
+
+This module provides various types for static type checking as described
```

Comment:
This documentation feels a bit thin. Perhaps we could have examples of WSGI code using some of these aliases.

Reply:
I agree about the examples, but when I procrastinated over two options, I gave up :)

1. Add an example to this section. But right below this section there is the "Example" section and this would feel like a duplication.
2. Add types to the example section. But in a way this distracts from the point of the example.